packages/office-ui-fabric-react/src/components/HoverCard/examples/HoverCard.EventListenerTarget.Example.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react';
+import { HoverCard, IExpandingCardProps } from 'office-ui-fabric-react/lib/HoverCard';
+import { DetailsList, buildColumns, IColumn } from 'office-ui-fabric-react/lib/DetailsList';
+import { DirectionalHint } from 'office-ui-fabric-react/lib/common/DirectionalHint';
+import { Fabric } from 'office-ui-fabric-react/lib/Fabric';
+import { createListItems, IExampleItem } from 'office-ui-fabric-react/lib/utilities/exampleData';
+import { KeyCodes } from '@uifabric/utilities';
+import { mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
+import { IconButton } from 'office-ui-fabric-react';
+
+const classNames = mergeStyleSets({
+  compactCard: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    height: '100%'
+  },
+  expandedCard: {
+    padding: '16px 24px'
+  },
+  item: {
+    selectors: {
+      '&:hover': {
+        textDecoration: 'underline',
+        cursor: 'pointer'
+      }
+    }
+  }
+});
+
+interface IHoverCardFieldProps {
+  expandingCardProps: IExpandingCardProps;
+}
+
+interface IHoverCardFieldState {
+  contentRendered?: boolean;
+}
+
+class HoverCardField extends React.Component<IHoverCardFieldProps, IHoverCardFieldState> {
+  private targetElementRef: React.RefObject<HTMLDivElement> = React.createRef<HTMLDivElement>();
+  private eventListenerRef: React.RefObject<HTMLElement> = React.createRef<HTMLElement>();
+
+  constructor(props: IHoverCardFieldProps) {
+    super(props);
+
+    this.state = {
+      contentRendered: false
+    };
+  }
+
+  public componentDidMount() {
+    this.setState({ contentRendered: true });
+  }
+
+  public render() {
+    return (
+      <div ref={this.targetElementRef} data-is-focusable={true}>
+        {typeof this.props.children === 'function' && this.props.children(this.eventListenerRef)}
+        {this.state.contentRendered && (
+          <HoverCard
+            expandingCardProps={this.props.expandingCardProps}
+            target={this.targetElementRef.current}
+            eventListenerTarget={this.eventListenerRef.current}
+            cardDismissDelay={300}
+            onCardVisible={this._log('onCardVisible')}
+            onCardHide={this._log('onCardHide')}
+            trapFocus={true}
+            openHotKey={KeyCodes.enter}
+          />
+        )}
+      </div>
+    );
+  }
+
+  private _log(text: string): () => void {
+    return (): void => {
+      console.log(text);
+    };
+  }
+}
+
+export class HoverCardEventListenerTargetExample extends React.Component<{}, {}> {
+  private _items: IExampleItem[] = createListItems(10);
+  private _columns: IColumn[] = this._buildColumns();
+
+  public render() {
+    return (
+      <Fabric>
+        <p>
+          Using the target to tag hover card on the right side of first column, and using eventListenerTarget to launch the card only when
+          hover over text field, hover over icon doesn't trigger card open.
+        </p>
+        <DetailsList
```

Comment:
I am not sure if we will feel comfortable to have another example containing a `DetailsList` on the HoverCard page. It starts to get heavy and perhaps to demo this new feature we could use something very simple. In the future we would probably need to refactor the rest of the examples too.

Reply:
I agree, I plan to have a simple example at first, but changed to `DeatilsList` afterwards thinking better to be consisten with exisiting example patterns.